pytools/persistent_dict.py

```diff
@@ -460,7 +460,7 @@ def update_for_time(self, key_hash: Hash, key: Any) -> None:
         import datetime
 
         # Convert to datetime object
-        self.rec(key_hash, datetime.datetime.combine(datetime.date.today(), key))
```

Comment:
Well, it *was* a deterministic hash for exactly one day. :rofl:

Reply:
We have a ‘wall of bugs’ in the office, this will go on it.